index.html

```diff
@@ -0,0 +1,167 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Pacifico&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css?family=Varela+Round&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="icon" href="/images/favicon.ico" />
+    <link rel="stylesheet" type="text/css" href="style.css" />
+    <title>Baseball Project</title>
+  </head>
+
+  <body>
+    <section>
+      <div class="image-box">
+        <img src="images/vanilla_coding_logo.png" />
+      </div>
+      <h1>Baseball</h1>
+
+      <!-- 야구게임 Start -->
+      <script>
+    
+      const initAnswer = () => {
+        const flags = Array(10).fill(false);
+      
+        let answer = '';
+        while (answer.length < 3) {
+          const random = parseInt(Math.random() * 10);
+      
+          if (!flags[random]) {
+            answer += random;
+            flags[random] = true;
+          }
+        }
+        return answer;
+      }
+      
+      const gameObject = {
+        answer: initAnswer(),
+        attempts: 0,
+        isClear: false,
+        end: false
+      };
+      
+      const makeHintStr = (number, hint) => {
+        return `<strong>시도 ${gameObject.attempts}</strong>: ${number}, <strong>결과</strong>: ${hint}`;
+      }
```

Comment:
```suggestion
      };
```
이 부분도 함수 표현식입니다.
이후 같은 코멘트는 생략하도록 하겠습니다.